keyboards/3.0/pgd-Khar-t-k0-qwerty.xml

```diff
@@ -59,13 +59,13 @@
       <key id="alt_k"      output="\u{10A32}\m{A}" />
       <key id="K"          output="\u{10A11}\m{A}" />
       <key id="q"          output="\u{10A11}\m{A}" />
-      <key id="Q"          output="\u{10A11}\m{A}" />
+      <key id="Q"          output="\u{10A11}\m{A}" /> <!--intentional duplicate -->
```

Comment:
Perhaps something such as "q and Q produce the same output"?

For that matter, why not just use the same key in two spots in the layout?